rspec-core/lib/rspec/core/runner.rb

```diff
@@ -130,6 +130,12 @@ def configure(err, out)
         @configuration.error_stream = err
         @configuration.output_stream = out if @configuration.output_stream == $stdout
         @options.configure(@configuration)
+
+        unless @configuration.warnings_set?
+          RSpec.warning "config.warnings isn't set. Please set it to either `:all`, `:deprecations_only`, or " \
+                        "`:none`. Setting it to at least `:deprecations_only` is highly" \
+                        "recommended to receive advance notice of future Ruby breaking changes."
```

Comment:
```suggestion
                        "recommended to receive advance notice of future Ruby breaking changes.",
                        :call_site => nil
```

This was a bit tricky.
So, when we run our cucumber tests (/features), specifically those two that `require "rspec/autorun"` issue the warning that "config.warnings isn't set".
It then attempts to find a first line where this was triggered outside RSpec's own code, and fails.
This fixed `cucumber` in rspec-core for me locally.
Sorry for the inconvenience, and thanks for the new feautre!

Reply:
Good catch